scripts/Interface/ui/sans_isis/SANSSaveOtherWindow.py

```diff
@@ -0,0 +1,121 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+from . import ui_save_other_dialog as ui_save_other_dialog
+from PyQt4 import QtGui, QtCore
+from sans.common.enums import SaveType
+from mantidqtpython import MantidQt
+
+try:
```

Comment:
Can we remove this `try...except` block? I don't think it's used and it will make porting to the new workbench easuer

Reply:
Ah I see. I missed the `*` import. There's not a way to do that yet but there soon will be. Can you change the `*` import to `from mantidplot import pymantidplot` so it's easier to search for?